### ISSUE TITLE
Changes needed to host fonts on GF

### ISSUE THREAD

**Issue by m4rc1e** [supertype-de/Blinker#1]
I've taken a quick look at the fonts and we'll need to change the following:

- SemiLight needs to be renamed to ExtraLight. Our API doesn't support SemiLight styles
- OS/2 usWeightclasses: Light should be 300, SemiBold should be 600, 
- Copyright should be "Copyright 2019 The Blinker Project Authors (https://github.com/supertype-de/Blinker/issues/new)"

Once this is done, I'll add a build script and push it to google/fonts.

**Comment by supertype-de** [comment]
@m4rc1e we did the fixes

**Comment by m4rc1e** [comment]
Unfortunately, you've introduced more errors. The previous fonts had 55 fails for the family, the new pr has 63.

## Fontbakery report

Fontbakery version: 0.7.6

<details>
<summary><b>[3] Family checks</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Fonts have equal glyph names?</summary>

* [com.google.fonts/check/family/equal_glyph_names](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/family/equal_glyph_names)
* 🔥 **FAIL** Glyphname 'uni2117' is defined on /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Thin.ttf but is missing on /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Black.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Bold.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-ExtraBold.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-ExtraLight.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Headline.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Light.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Regular.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-SemiBold.ttf.
* 🔥 **FAIL** Glyphname 'uni2120' is defined on /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Black.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Bold.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-ExtraBold.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-ExtraLight.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Headline.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Light.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Regular.ttf, /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-SemiBold.ttf but is missing on /Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Thin.ttf.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check font has a license.</summary>

* [com.google.fonts/check/family/has_license](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/family/has_license)
* 🔥 **FAIL** No license file was found. Please add an OFL.txt or a LICENSE.txt file. If you are running fontbakery on a Google Fonts upstream repo, which is fine, just make sure there is a temporary license file in the same folder. [code: no-license]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Fonts have consistent underline thickness?</summary>

* [com.google.fonts/check/family/underline_thickness](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/post.html#com.google.fonts/check/family/underline_thickness)
* 🔥 **FAIL** Thickness of the underline is not the same accross this family. In order to fix this, please make sure that the underlineThickness value is the same in the 'post' table of all of this family font files.
Detected underlineThickness values are:
	/Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Black.ttf: 164
	/Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Bold.ttf: 113
	/Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-ExtraBold.ttf: 134
	/Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-ExtraLight.ttf: 29
	/Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Headline.ttf: 142
	/Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Light.ttf: 49
	/Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Regular.ttf: 63
	/Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-SemiBold.ttf: 87
	/Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Thin.ttf: 17


</details>
<br>
</details>
<details>
<summary><b>[7] Blinker-Black.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Check glyph coverage.</summary>

* [com.google.fonts/check/glyph_coverage](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage)
* 🔥 **FAIL** Missing required codepoints: 0x000D (CARRIAGE RETURN) and 0x2215 (DIVISION SLASH)

</details>
<details>
<summary>🔥 <b>FAIL:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
* 🔥 **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache: 'http://www.apache.org/licenses/LICENSE-2.0' or Open Font License: 'http://scripts.sil.org/OFL'
 For a small set of legacy families the Ubuntu Font License 'https://www.ubuntu.com/legal/terms-and-policies/font-licence' may be acceptable as well.
When in doubt, please choose OFL for new font projects. [code: no-license-found]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Copyright notices match canonical pattern in fonts</summary>

* [com.google.fonts/check/font_copyright](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/font_copyright)
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'

</details>
<details>
<summary>🔥 <b>FAIL:</b> Checking OS/2 fsSelection value.</summary>

* [com.google.fonts/check/fsselection](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fsselection)
* 🔥 **FAIL** OS/2 fsSelection REGULAR bit should be set. [code: bad-REGULAR]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_FAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/familyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/familyname)
* 🔥 **FAIL** Entry [FONT_FAMILY_NAME(1):MACINTOSH(1)] on the 'name' table: Expected 'Blinker' but got 'Blinker Black'.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_SUBFAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/subfamilyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/subfamilyname)
* 🔥 **FAIL** SUBFAMILY_NAME for Mac "Regular" must be "Black" [code: bad-familyname]

</details>
<details>
<summary>🔥 <b>FAIL:</b> PPEM must be an integer on hinted fonts.</summary>

* [com.google.fonts/check/integer_ppem_if_hinted](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/integer_ppem_if_hinted)
* 🔥 **FAIL** This is a hinted font, so it must have bit 3 set on the flags of the head table, so that PPEM values will be rounded into and integer value.

</details>
<br>
</details>
<details>
<summary><b>[4] Blinker-Bold.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Check glyph coverage.</summary>

* [com.google.fonts/check/glyph_coverage](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage)
* 🔥 **FAIL** Missing required codepoints: 0x000D (CARRIAGE RETURN) and 0x2215 (DIVISION SLASH)

</details>
<details>
<summary>🔥 <b>FAIL:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
* 🔥 **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache: 'http://www.apache.org/licenses/LICENSE-2.0' or Open Font License: 'http://scripts.sil.org/OFL'
 For a small set of legacy families the Ubuntu Font License 'https://www.ubuntu.com/legal/terms-and-policies/font-licence' may be acceptable as well.
When in doubt, please choose OFL for new font projects. [code: no-license-found]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Copyright notices match canonical pattern in fonts</summary>

* [com.google.fonts/check/font_copyright](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/font_copyright)
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'

</details>
<details>
<summary>🔥 <b>FAIL:</b> PPEM must be an integer on hinted fonts.</summary>

* [com.google.fonts/check/integer_ppem_if_hinted](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/integer_ppem_if_hinted)
* 🔥 **FAIL** This is a hinted font, so it must have bit 3 set on the flags of the head table, so that PPEM values will be rounded into and integer value.

</details>
<br>
</details>
<details>
<summary><b>[7] Blinker-ExtraBold.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Check glyph coverage.</summary>

* [com.google.fonts/check/glyph_coverage](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage)
* 🔥 **FAIL** Missing required codepoints: 0x000D (CARRIAGE RETURN) and 0x2215 (DIVISION SLASH)

</details>
<details>
<summary>🔥 <b>FAIL:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
* 🔥 **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache: 'http://www.apache.org/licenses/LICENSE-2.0' or Open Font License: 'http://scripts.sil.org/OFL'
 For a small set of legacy families the Ubuntu Font License 'https://www.ubuntu.com/legal/terms-and-policies/font-licence' may be acceptable as well.
When in doubt, please choose OFL for new font projects. [code: no-license-found]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Copyright notices match canonical pattern in fonts</summary>

* [com.google.fonts/check/font_copyright](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/font_copyright)
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'

</details>
<details>
<summary>🔥 <b>FAIL:</b> Checking OS/2 fsSelection value.</summary>

* [com.google.fonts/check/fsselection](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fsselection)
* 🔥 **FAIL** OS/2 fsSelection REGULAR bit should be set. [code: bad-REGULAR]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_FAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/familyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/familyname)
* 🔥 **FAIL** Entry [FONT_FAMILY_NAME(1):MACINTOSH(1)] on the 'name' table: Expected 'Blinker' but got 'Blinker ExtraBold'.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_SUBFAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/subfamilyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/subfamilyname)
* 🔥 **FAIL** SUBFAMILY_NAME for Mac "Regular" must be "ExtraBold" [code: bad-familyname]

</details>
<details>
<summary>🔥 <b>FAIL:</b> PPEM must be an integer on hinted fonts.</summary>

* [com.google.fonts/check/integer_ppem_if_hinted](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/integer_ppem_if_hinted)
* 🔥 **FAIL** This is a hinted font, so it must have bit 3 set on the flags of the head table, so that PPEM values will be rounded into and integer value.

</details>
<br>
</details>
<details>
<summary><b>[7] Blinker-ExtraLight.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Check glyph coverage.</summary>

* [com.google.fonts/check/glyph_coverage](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage)
* 🔥 **FAIL** Missing required codepoints: 0x000D (CARRIAGE RETURN) and 0x2215 (DIVISION SLASH)

</details>
<details>
<summary>🔥 <b>FAIL:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
* 🔥 **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache: 'http://www.apache.org/licenses/LICENSE-2.0' or Open Font License: 'http://scripts.sil.org/OFL'
 For a small set of legacy families the Ubuntu Font License 'https://www.ubuntu.com/legal/terms-and-policies/font-licence' may be acceptable as well.
When in doubt, please choose OFL for new font projects. [code: no-license-found]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Copyright notices match canonical pattern in fonts</summary>

* [com.google.fonts/check/font_copyright](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/font_copyright)
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'

</details>
<details>
<summary>🔥 <b>FAIL:</b> Checking OS/2 fsSelection value.</summary>

* [com.google.fonts/check/fsselection](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fsselection)
* 🔥 **FAIL** OS/2 fsSelection REGULAR bit should be set. [code: bad-REGULAR]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_FAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/familyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/familyname)
* 🔥 **FAIL** Entry [FONT_FAMILY_NAME(1):MACINTOSH(1)] on the 'name' table: Expected 'Blinker' but got 'Blinker ExtraLight'.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_SUBFAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/subfamilyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/subfamilyname)
* 🔥 **FAIL** SUBFAMILY_NAME for Mac "Regular" must be "ExtraLight" [code: bad-familyname]

</details>
<details>
<summary>🔥 <b>FAIL:</b> PPEM must be an integer on hinted fonts.</summary>

* [com.google.fonts/check/integer_ppem_if_hinted](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/integer_ppem_if_hinted)
* 🔥 **FAIL** This is a hinted font, so it must have bit 3 set on the flags of the head table, so that PPEM values will be rounded into and integer value.

</details>
<br>
</details>
<details>
<summary><b>[7] Blinker-Headline.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Checking file is named canonically.</summary>

* [com.google.fonts/check/canonical_filename](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/canonical_filename)
* 🔥 **FAIL** Style name used in "/Users/marcfoley/Type/upstream_families/Blinker/fonts_ttf/Blinker-Headline.ttf" is not canonical. You should rebuild the font using any of the following style names: "Thin", "ExtraLight", "Light", "Regular", "Medium", "SemiBold", "Bold", "ExtraBold", "Black", "Thin Italic", "ExtraLight Italic", "Light Italic", "Italic", "Medium Italic", "SemiBold Italic", "Bold Italic", "ExtraBold Italic", "Black Italic".

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check glyph coverage.</summary>

* [com.google.fonts/check/glyph_coverage](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage)
* 🔥 **FAIL** Missing required codepoints: 0x000D (CARRIAGE RETURN) and 0x2215 (DIVISION SLASH)

</details>
<details>
<summary>🔥 <b>FAIL:</b> Checking OS/2 usWeightClass.</summary>

* [com.google.fonts/check/usweightclass](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/usweightclass)
* 🔥 **FAIL** OS/2 usWeightClass expected value for '' is 400 but this font has 900.

</details>
<details>
<summary>🔥 <b>FAIL:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
* 🔥 **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache: 'http://www.apache.org/licenses/LICENSE-2.0' or Open Font License: 'http://scripts.sil.org/OFL'
 For a small set of legacy families the Ubuntu Font License 'https://www.ubuntu.com/legal/terms-and-policies/font-licence' may be acceptable as well.
When in doubt, please choose OFL for new font projects. [code: no-license-found]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Copyright notices match canonical pattern in fonts</summary>

* [com.google.fonts/check/font_copyright](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/font_copyright)
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_SUBFAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/subfamilyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/subfamilyname)
* 🔥 **FAIL** SUBFAMILY_NAME for Mac "Regular" must be "" [code: bad-familyname]

</details>
<details>
<summary>🔥 <b>FAIL:</b> PPEM must be an integer on hinted fonts.</summary>

* [com.google.fonts/check/integer_ppem_if_hinted](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/integer_ppem_if_hinted)
* 🔥 **FAIL** This is a hinted font, so it must have bit 3 set on the flags of the head table, so that PPEM values will be rounded into and integer value.

</details>
<br>
</details>
<details>
<summary><b>[10] Blinker-Light.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Check glyph coverage.</summary>

* [com.google.fonts/check/glyph_coverage](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage)
* 🔥 **FAIL** Missing required codepoints: 0x000D (CARRIAGE RETURN) and 0x2215 (DIVISION SLASH)

</details>
<details>
<summary>🔥 <b>FAIL:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
* 🔥 **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache: 'http://www.apache.org/licenses/LICENSE-2.0' or Open Font License: 'http://scripts.sil.org/OFL'
 For a small set of legacy families the Ubuntu Font License 'https://www.ubuntu.com/legal/terms-and-policies/font-licence' may be acceptable as well.
When in doubt, please choose OFL for new font projects. [code: no-license-found]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Copyright notices match canonical pattern in fonts</summary>

* [com.google.fonts/check/font_copyright](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/font_copyright)
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'

</details>
<details>
<summary>🔥 <b>FAIL:</b> Checking OS/2 fsSelection value.</summary>

* [com.google.fonts/check/fsselection](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fsselection)
* 🔥 **FAIL** OS/2 fsSelection REGULAR bit should be set. [code: bad-REGULAR]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_FAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/familyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/familyname)
* 🔥 **FAIL** Entry [FONT_FAMILY_NAME(1):MACINTOSH(1)] on the 'name' table: Expected 'Blinker' but got 'Blinker ExtraLight'.
* 🔥 **FAIL** Entry [FONT_FAMILY_NAME(1):WINDOWS(3)] on the 'name' table: Expected 'Blinker Light' but got 'Blinker ExtraLight'.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_SUBFAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/subfamilyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/subfamilyname)
* 🔥 **FAIL** SUBFAMILY_NAME for Mac "Regular" must be "Light" [code: bad-familyname]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FULL_FONT_NAME entries. </summary>

* [com.google.fonts/check/name/fullfontname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/fullfontname)
* 🔥 **FAIL** Entry [FULL_FONT_NAME(4):MACINTOSH(1)] on the 'name' table: Expected 'Blinker Light' but got 'Blinker ExtraLight'.
* 🔥 **FAIL** Entry [FULL_FONT_NAME(4):WINDOWS(3)] on the 'name' table: Expected 'Blinker Light' but got 'Blinker ExtraLight'.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: POSTSCRIPT_NAME entries. </summary>

* [com.google.fonts/check/name/postscriptname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/postscriptname)
* 🔥 **FAIL** Entry [POSTSCRIPT_NAME(6):MACINTOSH(1)] on the 'name' table: Expected 'Blinker-Light' but got 'Blinker-ExtraLight'.
* 🔥 **FAIL** Entry [POSTSCRIPT_NAME(6):WINDOWS(3)] on the 'name' table: Expected 'Blinker-Light' but got 'Blinker-ExtraLight'.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: TYPOGRAPHIC_SUBFAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/typographicsubfamilyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/typographicsubfamilyname)
* 🔥 **FAIL** TYPOGRAPHIC_SUBFAMILY_NAME for Win is incorrect. It must be "Light".
* 🔥 **FAIL** TYPOGRAPHIC_SUBFAMILY_NAME for Mac is incorrect. It must be "Light". Please note, this record can be safely deleted.

</details>
<details>
<summary>🔥 <b>FAIL:</b> PPEM must be an integer on hinted fonts.</summary>

* [com.google.fonts/check/integer_ppem_if_hinted](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/integer_ppem_if_hinted)
* 🔥 **FAIL** This is a hinted font, so it must have bit 3 set on the flags of the head table, so that PPEM values will be rounded into and integer value.

</details>
<br>
</details>
<details>
<summary><b>[4] Blinker-Regular.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Check glyph coverage.</summary>

* [com.google.fonts/check/glyph_coverage](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage)
* 🔥 **FAIL** Missing required codepoints: 0x000D (CARRIAGE RETURN) and 0x2215 (DIVISION SLASH)

</details>
<details>
<summary>🔥 <b>FAIL:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
* 🔥 **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache: 'http://www.apache.org/licenses/LICENSE-2.0' or Open Font License: 'http://scripts.sil.org/OFL'
 For a small set of legacy families the Ubuntu Font License 'https://www.ubuntu.com/legal/terms-and-policies/font-licence' may be acceptable as well.
When in doubt, please choose OFL for new font projects. [code: no-license-found]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Copyright notices match canonical pattern in fonts</summary>

* [com.google.fonts/check/font_copyright](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/font_copyright)
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'

</details>
<details>
<summary>🔥 <b>FAIL:</b> PPEM must be an integer on hinted fonts.</summary>

* [com.google.fonts/check/integer_ppem_if_hinted](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/integer_ppem_if_hinted)
* 🔥 **FAIL** This is a hinted font, so it must have bit 3 set on the flags of the head table, so that PPEM values will be rounded into and integer value.

</details>
<br>
</details>
<details>
<summary><b>[7] Blinker-SemiBold.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Check glyph coverage.</summary>

* [com.google.fonts/check/glyph_coverage](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage)
* 🔥 **FAIL** Missing required codepoints: 0x000D (CARRIAGE RETURN) and 0x2215 (DIVISION SLASH)

</details>
<details>
<summary>🔥 <b>FAIL:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
* 🔥 **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache: 'http://www.apache.org/licenses/LICENSE-2.0' or Open Font License: 'http://scripts.sil.org/OFL'
 For a small set of legacy families the Ubuntu Font License 'https://www.ubuntu.com/legal/terms-and-policies/font-licence' may be acceptable as well.
When in doubt, please choose OFL for new font projects. [code: no-license-found]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Copyright notices match canonical pattern in fonts</summary>

* [com.google.fonts/check/font_copyright](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/font_copyright)
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'

</details>
<details>
<summary>🔥 <b>FAIL:</b> Checking OS/2 fsSelection value.</summary>

* [com.google.fonts/check/fsselection](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fsselection)
* 🔥 **FAIL** OS/2 fsSelection REGULAR bit should be set. [code: bad-REGULAR]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_FAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/familyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/familyname)
* 🔥 **FAIL** Entry [FONT_FAMILY_NAME(1):MACINTOSH(1)] on the 'name' table: Expected 'Blinker' but got 'Blinker SemiBold'.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_SUBFAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/subfamilyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/subfamilyname)
* 🔥 **FAIL** SUBFAMILY_NAME for Mac "Regular" must be "SemiBold" [code: bad-familyname]

</details>
<details>
<summary>🔥 <b>FAIL:</b> PPEM must be an integer on hinted fonts.</summary>

* [com.google.fonts/check/integer_ppem_if_hinted](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/integer_ppem_if_hinted)
* 🔥 **FAIL** This is a hinted font, so it must have bit 3 set on the flags of the head table, so that PPEM values will be rounded into and integer value.

</details>
<br>
</details>
<details>
<summary><b>[7] Blinker-Thin.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Check glyph coverage.</summary>

* [com.google.fonts/check/glyph_coverage](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage)
* 🔥 **FAIL** Missing required codepoints: 0x000D (CARRIAGE RETURN) and 0x2215 (DIVISION SLASH)

</details>
<details>
<summary>🔥 <b>FAIL:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
* 🔥 **FAIL** A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry. Currently accepted licenses are Apache: 'http://www.apache.org/licenses/LICENSE-2.0' or Open Font License: 'http://scripts.sil.org/OFL'
 For a small set of legacy families the Ubuntu Font License 'https://www.ubuntu.com/legal/terms-and-policies/font-licence' may be acceptable as well.
When in doubt, please choose OFL for new font projects. [code: no-license-found]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Copyright notices match canonical pattern in fonts</summary>

* [com.google.fonts/check/font_copyright](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/font_copyright)
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'
* 🔥 **FAIL** Name Table entry: Copyright notices should match a pattern similar to: 'Copyright 2017 The Familyname Project Authors (git url)'
But instead we have got: 'Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)'

</details>
<details>
<summary>🔥 <b>FAIL:</b> Checking OS/2 fsSelection value.</summary>

* [com.google.fonts/check/fsselection](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fsselection)
* 🔥 **FAIL** OS/2 fsSelection REGULAR bit should be set. [code: bad-REGULAR]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_FAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/familyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/familyname)
* 🔥 **FAIL** Entry [FONT_FAMILY_NAME(1):MACINTOSH(1)] on the 'name' table: Expected 'Blinker' but got 'Blinker Thin'.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_SUBFAMILY_NAME entries. </summary>

* [com.google.fonts/check/name/subfamilyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/subfamilyname)
* 🔥 **FAIL** SUBFAMILY_NAME for Mac "Regular" must be "Thin" [code: bad-familyname]

</details>
<details>
<summary>🔥 <b>FAIL:</b> PPEM must be an integer on hinted fonts.</summary>

* [com.google.fonts/check/integer_ppem_if_hinted](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/integer_ppem_if_hinted)
* 🔥 **FAIL** This is a hinted font, so it must have bit 3 set on the flags of the head table, so that PPEM values will be rounded into and integer value.

</details>
<br>
</details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS |
|:-----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 63 | 34 | 534 | 55 | 530 |
| 0% | 5% | 3% | 44% | 5% | 44% |

**Note:** The following loglevels were omitted in this report:
* **WARN**
* **SKIP**
* **INFO**
* **PASS**

**Comment by supertype-de** [comment]
@m4rc1e you guys could have informed us earlier, that the errors where not tolerable then we could have worked on that earlier. In the latest version we pushed we simply changed the style names and values and the copyright entry.

**Comment by supertype-de** [comment]
Regarding the copyright entry: we do not understand what is required of us. Please explain why "Copyright 2019, Blinker project authors Juergen Huber and Martin Wenzel (https://github.com/supertype-de/Blinker)" is a fail

**Comment by m4rc1e** [comment]
We don't want individual contributors mentioned in the copyright string. We're looking for.

"Copyright 2019 The Blinker project authors (https://github.com/supertype-de/Blinker)" 

> you guys could have informed us earlier, that the errors where not tolerable then we could have worked on that earlier. In the latest version we pushed we simply changed the style names and values and the copyright entry.

My plan was to fix these myself :-) which I still plan to do. I just needed the more laborious stuff taken care of first.

**Comment by supertype-de** [comment]
Our License URL entry (ID 14) is "http://scripts.sil.org/OFL" this should thus not produce a fail, or?

Furthermore we don't understand how to fix "FAIL No license file was found". Do you require a reference to "https://github.com/supertype-de/Blinker/blob/master/LICENSE"?

**Comment by m4rc1e** [comment]
> Our License URL entry (ID 14) is "http://scripts.sil.org/OFL" this should thus not produce a fail, or?

> Furthermore we don't understand how to fix "FAIL No license file was found". Do you require a reference to "https://github.com/supertype-de/Blinker/blob/master/LICENSE"?

Yeah those look like false positives. Please ignore.

**Comment by supertype-de** [comment]
Underline thickness should vary. This should not produce a fail

**Comment by m4rc1e** [comment]
> 🔥 FAIL: Fonts have consistent underline thickness?

We don't want underline thicknesses to vary across a family. If you have different styles on the same line and they're underlined, it looks strange. I'm happy to ignore this though.

**Comment by supertype-de** [comment]
Yes, please ignore. I feel that makes no sense.

**Comment by supertype-de** [comment]
I'll take care of any missing glyph and the name changes

**Comment by supertype-de** [comment]
I did another round of adjustments (setting fsSelection, fixing glitch with uni2117 glyph name in Thin, editing font style names entries for ExtraLight and Light). Give us a sign, we hope this works for you now.

**Comment by supertype-de** [comment]
HI @m4rc1e , we hope you're good with the update. Let us know if need to fix anything else

**Comment by m4rc1e** [comment]
Do you have a description for this family?

I was going to use the ReadMe but it doesn't describe the family. For inspiration see any "About" section for a family hosted on GF e.g https://fonts.google.com/specimen/Cutive+Mono

**Comment by supertype-de** [comment]
Hi Marc,

we provided this one in january, perhaps it faded through time. :)
Will this do?




Best
Juergen

**Comment by supertype-de** [comment]
See https://github.com/supertype-de/Blinker/blob/master/DESCRIPTION

**Comment by m4rc1e** [comment]
Awesome thank you!

**Comment by supertype-de** [comment]
Will you give us a sign when the font is available via google? Thanks.

**Comment by supertype-de** [comment]
Hi Marc,

is there anything else you can help you with?

Best regards
Juergen







supertype.de <http://supertype.de/>
Prof. Jürgen Huber
+49 (0)179 39 66 327
@sprtyp <https://twitter.com/sprtyp>
#supertype.de

**Comment by m4rc1e** [comment]
@supertype-de We're good now. I've merged the pr over at google/fonts. I'll let you know if we have anything that needs resolving. Thanks for your work on this!

**Comment by supertype-de** [comment]
Thanks Marc!

Do you have an idea when Blinker will be available as a google font?

J

**Comment by supertype-de** [comment]
Hi Marc,

as I understand, you are all set now and the font is ready to be released.
Can you tell when this will be? Sorry if I sound impatient but it has been
january since we first got in touch with you google guys. :)

Best regards
Juergen





supertype.de <http://supertype.de/>
Prof. Jürgen Huber
+49 (0)179 39 66 327
@sprtyp <https://twitter.com/sprtyp>
#supertype.de

**Comment by supertype-de** [comment]
Hi Marc,

it is me again. I fully understand that you may not be in the position
to answer my question. But maybe you can pass me on to somebody
who can, please?

Thank you very much!
Juergen

**Comment by m4rc1e** [comment]
Hey,

Perhaps next week. The Eng team who push the fonts are currently on holiday.
